.mailmap

```diff
@@ -1238,6 +1238,7 @@ alijosephine <alijosephine@gmail.com>
 amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
 anca-mc <anca-mc@users.noreply.github.com>
 andreo <andrey.torba@gmail.com>
+anutosh491 <87052487+anutosh491@users.noreply.github.com>
 anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
```

Comment:
This is still not correct because it is listing you as two separate contributors.

Reply:
I tried removing the second option . The git diff I applied
```
--- a/.mailmap
+++ b/.mailmap
@@ -1238,8 +1238,7 @@ alijosephine <alijosephine@gmail.com>
 andreo <andrey.torba@gmail.com>
-anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
+anutosh491 <andersonbhat491@gmail.com> anutosh491 <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
```
After running `mailmap_check.py` , I got this message which I felt was not what we would want !
```
←[31mThis author is not included in the .mailmap file:←[0m
Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
←[31m
The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.←[0m
←[31m
For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#update-
mailmap←[0m
←[32m
The following authors will be added to the AUTHORS file at the time of
the next SymPy release.←[0m

        Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
        Siddhant Jain <siddhantashoknagar@gmail.com>
        Tirthankar Mazumder <63574588+wermos@users.noreply.github.com>
```

Reply:
And if I get rid of the first option using this 
```
index 308a6e7526..d80b9165a3 100644
--- a/.mailmap
+++ b/.mailmap
@@ -1238,8 +1238,7 @@ alijosephine <alijosephine@gmail.com>
 amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
-anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
+Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
```
I got this message, which again seems wrong !
```
←[31mThis author is not included in the .mailmap file:←[0m
anutosh491 <andersonbhat491@gmail.com>
←[31m
The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.←[0m
←[31mThe mailmap file was reordered←[0m
←[31m
For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#update-
mailmap←[0m
←[32m

```

Reply:
You are already in the AUTHORS file under this name and email address:
https://github.com/sympy/sympy/blob/0bc2587aef09ab150bd408b7cd9e08baf7b5fd9c/AUTHORS#L1112
Is that the name and email address that you want to have in the AUTHORS file? (I asked this question above and you didn't answer)

Reply:
Then the change that needs to be made is this:
```diff
diff --git a/.mailmap b/.mailmap
index b39173b..52d0c68 100644
--- a/.mailmap
+++ b/.mailmap
@@ -1239,6 +1239,7 @@ amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”
 anca-mc <anca-mc@users.noreply.github.com>
 andreo <andrey.torba@gmail.com>
 anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
+anutosh491 <andersonbhat491@gmail.com> anutosh491 <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
 bluebrook <perl4logic@gmail.com>
 carstimon <carstimon@gmail.com>
```
Each line begins with the name and email address that is in the AUTHORS file and then lists the other name and email address that has been used for some of your commits.

Reply:
Thanks for the help again ! Can't thank you enough , mailmap has been confusing me since a while now , but now I know how to deal with it in other prs too , i'll handle other too prs which need such fix like this one for consistency